handlers/authentication.go

```diff
@@ -144,7 +144,7 @@ func performOpenshiftAuthentication(w http.ResponseWriter, r *http.Request) bool
 
 	expiresOn := time.Now().Add(time.Second * time.Duration(expiresInNumber))
 
-	business, err := getBusiness(r)
+	business, err := business.Get(&api.AuthInfo{Token: token})
```

Comment:
I'm not 100% sure this line change is needed. I'm testing to see if this change is required.

Reply:
this is not needed. I will revert this change.